inventory/kits/models/KitHistoryModels.py

```diff
@@ -31,8 +29,17 @@ class History(models.Model):
     type = models.CharField(max_length=50, choices=HISTORY_TYPE)
     date = models.DateField()
```

Comment:
could we change this to a DateTimeField also? in that way we can see what time the entry is been made

can also consider adding auto_add, so it datetime is populated upon creation of entry